test/ModelContextProtocol-SemanticKernel.Tests/ClaudeToolsTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentAssertions;
+using Microsoft.SemanticKernel;
+using ModelContextProtocol.SemanticKernel.Extensions;
+
+namespace ModelContextProtocol.SemanticKernel.Tests;
+
+public sealed class ClaudeToolsTests
+{
+    [Fact]
+    public async Task ListTools()
+    {
+        // Arrange
+        var ct = TestContext.Current.CancellationToken;
+        // TODO: see how to mock file-system
```

Comment:
Good idea.

I can make a change using https://www.nuget.org/packages/System.IO.Abstractions

Or if you have time, you can make a new PR or include this into this one.